pydata/templates/pydata/generic_import_form.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base_nav_fixed.html" %}
```

Comment:
@pbanaszkiewicz This template can be moved to `workshops/templates` if you want it to.